docker/entrypoint_prod.sh

```diff
@@ -76,6 +76,9 @@ elif [ "$1" == "run_intermittents_commenter" ]; then
 elif [ "$1" == "update_bugscache" ]; then
     newrelic-admin run-program ./manage.py update_bugscache
 
+elif [ "$1" == "update_files_bugzilla_map" ]; then
+    newrelic-admin run-program ./manage.py update_files_bugzilla_map
```

Comment:
Where are the cron tasks and their intervals set up? Heroku had a viewer, I didn't find them in GCE.

Reply:
That's something cloudOps will do: https://treeherder.readthedocs.io/infrastructure/administration.html#adding-or-changing-scheduled-tasks-and-environment-variables

Reply:
@Archaeopteryx do you have a cron schedule in mind for this?

Reply:
Sheriffs merge every 6 or 12 hours at (x * 6 + 4) UTC hours and the task completes soon after. The initial run on this machines is maybe 2 minutes, subsequent ones faster. If you are asking for a full hour: 4 times a day at 5, 11, 17, 23 UTC.